### PR TITLE
Crowdloan Rewards batch initialization and staking rewards scripts update within bootstrap_pallets

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
Updated scripts to create on chain data to play/test with.

https://github.com/ComposableFi/composable/pull/1431 Merged here

### CODE REVIEW

File: composablejs/packages/oracle_setup/src/config.json [hunk 1]
```diff
@@ -12,14 +12,13 @@
     "value": "http://localhost:3001/price/"
   },
   "oracle_parameters": {
-    "assetId": 4,
+    "assetID": 4,
     "threshold": 10,
     "minAnswers": 1,
-    "maxAnswers": 3,
+    "maXAnswers": 3,
```
Comment (andresmechali) [inline composablejs/packages/oracle_setup/src/config.json:18]: typo on the `X`?

**Review by andresmechali — APPROVED** [review]

File: composablejs/packages/bootstrap_pallets/src/constants/config.json [hunk 2]
```diff
@@ -52,12 +53,37 @@
...
+        "131": {
+          "assetId": "131",
+          "maxRewards": "10000000000000000000",
+          "rewardRate": {
+            "period": "PerSecond",
+            "amount": "77160494000"
+          }
+        }
+      },
+      "locK": {
```
Comment (Dom-Roth) [inline composablejs/packages/bootstrap_pallets/src/constants/config.json:76]: nit: typo?

Comment (Dom-Roth) [inline composablejs/packages/bootstrap_pallets/src/constants/config.json:76]:
```suggestion
      "lock": {
```
nit: typo?

**Review by Dom-Roth — APPROVED** [review]
LGTM!

File: composablejs/packages/bootstrap_pallets/src/bootstrap/crowdloanRewards/index.ts [hunk 3]
```diff
@@ -1,6 +1,58 @@
...
+          api.createType("PalletCrowdloanRewardsModelsRemoteAccount", {
+            RelayChain: api.createType("AccountId32", account).toU8a()
+          }),
+          rewardsPerAccount,
+          vestingPeriod
+        ] as [PalletCrowdloanRewardsModelsRemoteAccount, u128, u32];
+      }
+    }) as [PalletCrowdloanRewardsModelsRemoteAccount, u128, u32][];
+
+    txCalls.push(api.tx.sudo.sudo(api.tx.crowdloanRewards.populate(batchAccounts)));
```
Comment (vimukthi-git) [inline composablejs/packages/bootstrap_pallets/src/bootstrap/crowdloanRewards/index.ts:44]: Is it possible for us to use the same script to populate the live chain with the crowdloan details?

**Review by vimukthi-git — APPROVED** [review]
LGTM

File: composablejs/packages/bootstrap_pallets/src/bootstrap/crowdloanRewards/index.ts (hunk 3 above)
Comment (saadjhk) [reply composablejs/packages/bootstrap_pallets/src/bootstrap/crowdloanRewards/index.ts:44]: can be used with some modifications as rewards are constant/same for all atm

**Pull request merged**